core/string/lines_spec.rb

```diff
@@ -10,4 +10,12 @@
     ary = "hello world".send(@method, ' ')
     ary.should == ["hello ", "world"]
   end
+
+  ruby_version_is '2.4' do
+    context "when `chomp` keyword argument is passed" do
+      it "removes carriage return characters" do
+        "hello \nworld\n".lines(chomp: true).should == ["hello ", "world"]
+      end
+    end
+  end
```

Comment:
Since the shared spec below is already included for String#lines, this spec is duplicating the shared one. Could you remove it?

Reply:
@eregon Shared spec checks `lines` call with block, that's why I added a case for call without block. But OK

Reply:
Oh, I missed that. Then indeed it makes sense to test that case too.
Could you add it back? I am sorry.

Reply:
@eregon done